source/libnormaliz/output.cpp

```diff
@@ -1468,7 +1468,7 @@ void Output<Integer>::write_files() {
         }
         if (homogeneous && Result->isComputed(ConeProperty::IsSerreR1)) {
             if (Result->isIntegrallyClosed()) {
-                out << "original monoid satifies Serre cindition R1" << endl;
+                out << "original monoid satisfies Serre cindition R1" << endl;
             }
             else {
                 out << "original monoid vioaltes Serre cindition R1" << endl;
```

Comment:
Two more misspelled words appear in the context of this patch: "vioaltes" on line 1474 should be "violates".  The same misspelling occurs in test/test-grb/latt_ideal_grad.ref.  Also, all of the "cindition" instances visible here should be "condition".

Reply:
Thanks for noticing these!  I've just pushed two commits fixing these additional typos.